benchmarks-api/src/test/java/uk/co/real_logic/benchmarks/remote/LoadTestRigTest.java

```diff
@@ -302,19 +294,31 @@ void sendStopsWhenTotalNumberOfMessagesIsReached(final @TempDir Path tempDir)
             out,
             histogram,
             MINIMUM_NUMBER_OF_CPU_CORES * 2,
-            messageRecorder -> messageTransceiver);
+            messageRecorder ->
+            {
+                doAnswer(invocation ->
```

Comment:
Better to indent the lambda body so it stands out.

Reply:
Indented inner lambda.